catalogapp/src/deb/resources/etc/georchestra/catalogapp/log4j/log4j.properties

```diff
@@ -0,0 +1,11 @@
+log4j.rootLogger=WARN, R
+
+log4j.logger.catalogapp=WARN, R
+
+log4j.appender.R = org.apache.log4j.rolling.RollingFileAppender
+log4j.appender.R.RollingPolicy = org.apache.log4j.rolling.TimeBasedRollingPolicy
+log4j.appender.R.RollingPolicy.FileNamePattern = /tmp/catalogapp.%d.log.gz
+log4j.appender.R.RollingPolicy.ActiveFileName = /tmp/catalogapp.log
```

Comment:
This is a convenient directory where we are almost ensured to be able to write into, I think every other webapps follow the same convention.

I already thought about a better one created by the packages using a postinst script, but we have no way of figuring out which user would need to write into the directory, and chmod 777 was not an option)